cli/azd/pkg/apphost/manifest.go

```diff
@@ -49,6 +49,19 @@ type Resource struct {
 	// Some resources just represent connections to existing resources that need not be provisioned.  These resources have
 	// a "connectionString" property which is the connection string that should be used during binding.
 	ConnectionString *string `json:"connectionString,omitempty"`
+
+	// Dapr is present on dapr.v0 resources.
+	Dapr *DaprResourceMetadata `json:"dapr,omitempty"`
```

Comment:
Just checking - my reading of the PR for the manifest changes makes it seem like all these properties are actually siblings of the `type` property, instead of nested under a `dapr` property on the root resource in the manifest.  However, the way you have these model types written today, it expects the dapr node.

Which is correct?

Reply:
Ah, I may have forgotten to regenerate the repo's `.manifest`.  (AZD CLI always uses a freshly generated one by publishing the project.) The intent is that there is a separate `dapr` property that has all the Dapr-specific metadata, mostly because Golang serialization is so ugly and seems prone to collisions.  I suppose I could use `dapr` prefixes on properties in the manifest, but that makes the manifest ugly.  That said, I'm ok going with whichever the team prefers.

Reply:
I do see now that the right thing is happening - I'm fine with either form.  I sort of prefer the nesting that you have here because it makes it clearer in the model types what is set when, but if the aspire folks want all the properties at top level on the manifest, I'm happy with that approach as well.